docs/introduction/installation.md

```diff
@@ -0,0 +1,165 @@
+# Install Marko
+
+> [!NOTE]
+> If you'd like to experiment and learn about Marko directly in your browser, head over to our [Playground](/playground). You can develop a Marko application right there without any local setup.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en) `v18`, `v20`, `v22` or higher.
+- IDE - We recommend [VS Code](https://code.visualstudio.com/) and our [Official Marko extension](https://marketplace.visualstudio.com/items?itemName=Marko-JS.marko-vscode)
+- Terminal - You will use the command-line interface (CLI) to set up and start the application.
+
+## Marko Run Setup (Recommended)
+
+[Marko Run](https://github.com/marko-js/run) makes it easy to get started with little to no config and is the recommended starting point for a new Marko project. It's the
+official application framework from the Marko team and powered by [Vite](https://vite.dev/).
```

Comment:
Usually markdown gets confused if you split into multiple lines like this. Not sure if it was your doing or ours, but this should be merged into one line I think.

Reply:
Thanks, Fixed